lite/jupyterlite_xeus_python/build.py

```diff
@@ -39,6 +39,17 @@
 PLATFORM = "emscripten-wasm32"
 DEFAULT_REQUEST_TIMEOUT = 1  # in minutes
 
+# check if we are in a local build via
+# export XEUS_PYTHON_LOCAL_BUILD=1
+# export XEUS_PYTHON_BUID_DIR=$XEUS_PYTHON_BUILD_DIR
+XEUS_PYTHON_LOCAL_BUILD=False
+if os.environ.get("XEUS_PYTHON_LOCAL_BUILD") == "1":
+    XEUS_PYTHON_LOCAL_BUILD=True
+    XEUS_PYTHON_BUILD_DIR=os.environ.get("XEUS_PYTHON_BUILD_DIR")
+    if XEUS_PYTHON_BUILD_DIR is None:
+        raise RuntimeError("XEUS_PYTHON_BUILD_DIR is not set")
+    XEUS_PYTHON_BUILD_DIR=Path(XEUS_PYTHON_BUILD_DIR)
```

Comment:
Could we make the build directory as part of the Python package data (local dir installed alongside the Python package under site-packages)?